test/e2e/performanceprofile/functests/7_performance_kubelet_node/cgroups.go

```diff
@@ -76,6 +171,326 @@ var _ = Describe("[performance] Cgroups and affinity", Ordered, func() {
 			Expect(err).ToNot(HaveOccurred())
 			Expect(ovsLoadBalancing).To(Equal("0"))
 		})
+		It("[test_id:64098] ovs-vswitchd and ovsdb-server are under ovs.slice cgroup", func() {
+			processCgroups := "cpuset:/ovs.slice"
+			pidList, err := getOVSServicesPid(workerRTNode)
+			Expect(err).ToNot(HaveOccurred())
+			for _, pid := range pidList {
+				cgroupPath := fmt.Sprintf("/proc/%s/cgroup", pid)
+				checkCgroupCmd := []string{"cat", cgroupPath}
+				cgroupInfo, err := nodes.ExecCommandOnNode(checkCgroupCmd, workerRTNode)
+				ok, err := regexp.MatchString(processCgroups, cgroupInfo)
+				if err != nil {
+					testlog.Error(err.Error())
+				}
+				if ok {
+					testlog.Info(processCgroups)
+				}
+			}
+		})
+	})
+
+	Describe("Affinity", func() {
+		Context("ovn-kubenode Pods affinity ", func() {
+			BeforeEach(func() {
+				initialProfile = profile.DeepCopy()
+			})
+			It("[test_id:64100] matches with ovs process affinity", func() {
+				ovnPod, err := getOvnPod(workerRTNode)
+				Expect(err).ToNot(HaveOccurred(), "Unable to get ovnPod")
+				ovnContainersids, err := getOvnPodContainers(&ovnPod)
+				Expect(err).ToNot(HaveOccurred())
+				// Generally there are many containers inside a kubenode pods
```

Comment:
I think we actually care about a specific container. Only one of them has the pinning logic inside. I just forgot which one it is.